src/Components/ShowMore.jsx

```diff
@@ -0,0 +1,71 @@
+const ShowMore = ({ data }) => {
+
+    let percent = ((data.codewars.current.total / data.codewars.goal.total) * 100).toFixed(2)
+   
+    return (
+
+    <div className="additional-info">
+        <div>
+    <span className="codewar"><h3>CodeWars:</h3></span>
+    <p>
+        <span>Current Total: </span>
+        {data.codewars.current.total}
+    </p>
+    <p>
+        <span>Last Week: </span>
+        {data.codewars.current.lastWeek}
+    </p>
+    <p>
+        <span>Goal: </span>
+        {data.codewars.goal.total}
+    </p>
+    <p>
+        <span>Precent Of Goal Achieved: </span>
+       <div className={percent >= 100? "a" : percent > 50 ? "b": "c"}>{percent}%</div> 
+    </p>
+    </div>
+    <div>
+    <p>
+        <span className="score"><h3>Scores:</h3></span>
+    </p>
+    <p>
+        <span>Assignments: </span>
+        {data.cohort.scores.assignments}%
```

Comment:
The scores all seem to be off by a factor of 100, (0.78% instead of 78%), should multiple by 100 to get the right number